### PR TITLE
fix: add CLI devscript flag

### LINKED ISSUE #78: Add watch command to dev script from CLI prompt
```
  // package.json
  "scripts": {
    "dev": "next dev & npx next-video sync -w",
  },
```

### PR DESCRIPTION
**Pull request by luwes**
closes #78

### CODE REVIEW

File: src/cli/init.ts [hunk 1]
```diff
@@ -193,27 +201,53 @@ export async function handler(argv: Arguments) {
   // we should ask just in case.
   if (!ts) {
     ts = await confirm({ message: 'Is this a TypeScript project?', default: true });
+  }
 
-    if (ts) {
-      await createTSFile(path.join(process.cwd(), 'video.d.ts'));
-      changes.push([log.add, `Created video.d.ts.`]);
-    }
+  if (ts) {
```
Comment (luwes) [inline src/cli/init.ts:206]: moved this out of the `if (!ts) {}` scope which was a bug if using the flag I think

**Review by mmcc — APPROVED** [review]
This looks good to me, but do we want to include any kind of messaging around "You might want to add the watch command blah blah"?

**Comment by luwes** [comment]
@mmcc do you mean if the user chooses "no" or even before the question give more information on what it does?

**Comment by mmcc** [comment]
No I meant just as an instructional bit at the end. If the user said no, should we let them know at the end that it's a step they might want to add?

**Pull request merged**